README.md

```diff
@@ -29,8 +29,57 @@ This reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools)
 within the field of raman into a standardized representation using the
 [NeXus](https://www.nexusformat.org/) application definition [NXraman](https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXraman.html#nxraman).
 
-## Docs
-Extensive documentation of this pynxtools plugin is available [here](https://fairmat-nfdi.github.io/pynxtools-raman/). You can find information about getting started, how-to guides, the supported file formats, how to get involved, and much more there.
+
+## Step-by-Step Example
+Download the repository via git clone:
+```shell
+git clone https://github.com/FAIRmat-NFDI/pynxtools-raman.git
+```
+Switch to the project root folder:
+```shell
+cd pynxtools-raman
+```
+You see 3 Folders:
+- examples: contains example datasets, to show how the data conversion is done (currently 1 example from WITec and 1 example from the Raman Open Database)
+- tests: contains a test procedure and files, which are required for software development
+- src/pynxtools_raman: contains the source files, which contain the sub-reader function for Raman experiments. This only works in combination with the Python package [pynxtools](https://github.com/FAIRmat-NFDI/pynxtools). This contains the [Multiformat Reader](https://fairmat-nfdi.github.io/pynxtools/how-tos/use-multi-format-reader.html) and the respective sub-reader functions for WITec or the Raman Open Database. This also contains the config.json files in src/pynxtools_raman/config, which are necessary to map the data via the Multiformat Reader, by as well allowing individual adjustments. In this way each laboratory is able to map the data via the same reader, while each laboratory has its own individual electronic lab notebook structure.
+
+Consider setting up an invididual python environment, to seperate the python fuctnionalities of this package from the python funtionalities of your operating system:
+For Ubuntu-based systems:
+```shell
+python -m venv .pyenv
+source .pyenv/bin/activate
+```
+Verify its location via:
+```shell
+which python
+```
+It should point to the python folder, you created above with the name `.pyenv`.
+
+
+Install the python package:
+```shell
+pip install .
+```
+**Perform a data conversion**
+for the WITec dataset via:
+```shell
+dataconverter examples/witec/txt/eln_data.yaml examples/witec/txt/Si-wafer-Raman-Spectrum-1.txt src/pynxtools_raman/config/config_file_witec.json --reader raman --nxdl NXraman --output new_witec_example_neuxs.nxs
+```
+
+and for the Raman Open Database dataset set via:
+```shell
+dataconverter examples/database/rod/rod_file_1000679.rod src/pynxtools_raman/config/config_file_rod.json --reader raman --nxdl NXraman --output new_rod_example_neuxs.nxs
+```
+
+**For Example for the Raman Open Database command:**
+- You assign the reader name via `--reader raman`.
+- You assign the NeXus definition language (nxdl) via `--nxdl NXraman`.
```

Comment:
You don't assing the NeXus definition language, you define which application definition that is _described_ in nxdl to use.

Reply:
ah yea, all right. Adjusted it.